modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/upload/DDMFormUploadFileEntryHandler.java

```diff
@@ -126,6 +138,77 @@ protected Folder addFolder(long groupId, long userId)
 			serviceContext);
 	}
 
+	private User _createDDMFormGuestUser(long companyId)
+		throws PortalException {
+
+		long creatorUserId = 0;
+		boolean autoPassword = true;
+		String password1 = StringPool.BLANK;
+		String password2 = StringPool.BLANK;
+		boolean autoScreenName = false;
+		String screenName = DDMFormConstants.DDM_FORM_GUEST_USER_SCREEN_NAME;
+
+		Company company = _companyLocalService.getCompany(companyId);
+
+		String emailAddress = StringBundler.concat(
+			screenName, StringPool.AT, company.getMx());
+
+		Locale locale = LocaleUtil.getDefault();
+		String firstName = DDMFormConstants.DDM_FORM_GUEST_USER_FIRST_NAME;
+		String middleName = StringPool.BLANK;
+		String lastName = DDMFormConstants.DDM_FORM_GUEST_USER_LAST_NAME;
+		long prefixId = 0;
+		long suffixId = 0;
+		boolean male = true;
+		int birthdayMonth = Calendar.JANUARY;
+		int birthdayDay = 1;
+		int birthdayYear = 1970;
+		String jobTitle = StringPool.BLANK;
+		long[] groupIds = null;
+		long[] organizationIds = null;
+		long[] roleIds = null;
+		long[] userGroupIds = null;
+		boolean sendEmail = false;
+		ServiceContext serviceContext = null;
+
+		User user = _userLocalService.addUser(
+			creatorUserId, companyId, autoPassword, password1, password2,
+			autoScreenName, screenName, emailAddress, locale, firstName,
+			middleName, lastName, prefixId, suffixId, male, birthdayMonth,
+			birthdayDay, birthdayYear, jobTitle, groupIds, organizationIds,
+			roleIds, userGroupIds, sendEmail, serviceContext);
+
+		_userLocalService.updateStatus(
+			user.getUserId(), WorkflowConstants.STATUS_INACTIVE,
```

Comment:
@carolmariaabb, have you set this user as Inactive to avoid display it on the User's screen?

Reply:
@natocesarrego Yes, that was the intention. Do you think it is not necessary?

Reply:
Makes sense to me, @carolmariaabb.